apps/meteor/client/components/FilterByText.tsx

```diff
@@ -27,7 +27,7 @@ const FilterByText: FC<FilterByTextProps> = ({ placeholder, onChange: setFilter,
 	}, []);
 
 	useEffect(() => {
-		setFilter({ text });
```

Comment:
Isn’t `setFilter` a required prop? It should never be undefined, if it is happening, we should fix the place that is not setting the prop

Or make it optional.